test/validation/api/timer/timer.c

```diff
@@ -2918,9 +2919,11 @@ static void timer_test_periodic(odp_queue_type_t queue_type, int use_first, int
 		num_tmo = 0;
 		done = 0;
 
-		tmo = odp_timeout_alloc(pool);
-		ev  = odp_timeout_to_event(tmo);
-		CU_ASSERT_FATAL(ev != ODP_EVENT_INVALID);
+		if (!reuse_event || round == 0) {
+			tmo = odp_timeout_alloc(pool);
+			ev  = odp_timeout_to_event(tmo);
+			CU_ASSERT_FATAL(ev != ODP_EVENT_INVALID);
```

Comment:
Perhaps it would be safer to run the assert every round (or add another fatal check to the end of the loop), since it looks like `ev` could be `ODP_EVENT_INVALID` in case of failure timeout in the final while loop.

Reply:
Assert every round in v2.